### ISSUE TITLE
Syntax errors in the pm.sendRequest() script causing app to crash

### ISSUE THREAD

**Issue by DannyDainton** [a85/POSTMan-Chrome-Extension#6651]
**Describe the bug**

I was using the `pm.sendRequest()` function in one of my requests, my script was a bit rubbish and had issues within it but I didn't know that when I sent the request. After hitting the `Send` button the application just crashed. 

After reducing down the code I was using, the snippet below is the code I used to reproduce the problem, I'm sure it could be narrowed down more but I didn't investigate that far.

**To Reproduce**

1. Open the App
2. Create a new request ( `GET https://postman-echo.com/get`)
3. Add the following code to the `Pre-request Script`

```javascript
pm.sendRequest({
    url: 'https://api.fake.com',
    method: 'POST',
    body: {
        mode: 'raw',
        raw: { 
                "something": { 
                    "something": "else"
                } 
            }
        }
    }, (err, res) => {
        console.log({ err, res });
});
```

4. Send Request

**Expected behaviour**

Upon sending the request, the application should gracefully handle the error in the `pm.sendRequest()` function. Some relevant feedback to the user should be presented, rather than crashing the application.

**App information**
 - Native App
 - Postman Version 7.1.1
 - OS: macOS Mojave 10.14.5

**Additional context**

Extracted the followed error from the logs:

```bash
[64726][1559637623122][main][error]["Main~handleUncaughtError - Uncaught errors",{"name":"TypeError","message":"First argument must be a string or Buffer","stack":"TypeError: First argument must be a string or Buffer\n 
   at write_ (_http_outgoing.js:655:11)\n 
   at ClientRequest.write (_http_outgoing.js:630:10)\n 
   at Request.write (/Applications/Postman.app/Contents/Resources/app/node_modules/postman-request/request.js:1725:27)\n 
   at end (/Applications/Postman.app/Contents/Resources/app/node_modules/postman-request/request.js:628:18)\n 
   at Immediate._onImmediate (/Applications/Postman.app/Contents/Resources/app/node_modules/postman-request/request.js:657:7)\n 
   at runCallback (timers.js:781:20)\n 
   at tryOnImmediate (timers.js:743:5)\n 
   at processImmediate [as _immediateCallback] (timers.js:714:5)"}]
```

**Comment by vikiCoder** [comment]
Similar to this issue: https://github.com/postmanlabs/postman-app-support/issues/6180

**Comment by vikiCoder** [comment]
This issue is fixed in the [latest Postman App](https://www.getpostman.com/downloads/) release (v7.2.1). Passing non-string data into the raw body will not crash the app now.

**Comment by codenirvana** [comment]
This is fixed in the latest version of Postman app (v7.3.4).
